build.gradle.kts

```diff
@@ -268,6 +279,31 @@ subprojects {
                         }
                     }
                 }
+
+                configurationsContainer.register("screenshot") {
+                    reportSkippedTests = false
+                    reportFlakyTests = false
+                    enableDeviceDebug = true
+                    filter = "screenshot"
+
+                    targetsContainer.register("api28") {
+                        deviceName = "API28"
+
+                        scheduling {
+                            quota {
+                                retryCount = 1
+                                minimumSuccessCount = 1
+                            }
+
+                            testsCountBasedReservation {
+                                device = emulator28
+                                maximum = 1
+                                minimum = 1
+                                testsPerEmulator = 1
```

Comment:
Why so few? Are there any implicit restrictions for screenshot tests?

Reply:
Change strategy to static with one emulator